DESCRIPTION

```diff
@@ -44,8 +44,7 @@ Suggests:
 Config/testthat/edition: 3
 License: MIT + file LICENSE
 Imports:
-    Rcpp,
-    RcppEigen
+    Rcpp
```

Comment:
```suggestion
    Rcpp,
    RcppEigen
```

Reply:
I removed it from `Imports` on purpose as R CMD check flagged it should only be in `LinkingTo`. Indeed, it's never used in the R code, but only in the Rcpp code.